.github/workflows/go.yml

```diff
@@ -0,0 +1,34 @@
+name: Go
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build:
+    strategy:
+      matrix: 
+        go-version: [1.12.x, 1.13.x, 1.14.x]
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
```

Comment:
nit: extra empty line

Reply:
Had used the github gui to init so the emptylines came with the template I suspect. removed empty line